uniffi_bindgen/src/bindings/python/templates/RustBufferBuilder.py

```diff
@@ -97,7 +97,7 @@ def writeF64(self, v):
     {% when Type::Boolean -%}
 
     def writeBool(self, v):
-        self._pack_into(1, ">b", 0 if v else 1)
+        self._pack_into(1, ">b", 1 if v else 0)
```

Comment:
this is an unrelated bugfix, right?

Reply:
Oh, yes, yes it is - but one that I only happened to discover in the process of getting all this working.